code/datums/station_traits/negative_traits.dm

```diff
@@ -174,3 +174,26 @@
 		/// The bot's language holder - so we can randomize and change their language
 		var/datum/language_holder/bot_languages = found_bot.get_language_holder()
 		bot_languages.selected_language = bot_languages.get_random_spoken_language()
+
+/datum/station_trait/united_budget
+	name = "United Department Budget Management"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	show_in_report = TRUE
+	report_message = "As your station is selected for our financial experiments, all station budgets are united into one, and all budget card will be linked to that one."
```

Comment:
Your station has been selected for one of our financial experiments! All station budgets have been united into one, and all budget cards will be linked to one account!